lib/associations/base.js

```diff
@@ -1,3 +1,21 @@
-var Association = function () {};
+'use strict';
 
-module.exports = Association;
\ No newline at end of file
+var Association = function() {};
+
+// Normalize input - may be array or single obj, instance or primary key - convert it to an array of built objects
+Association.prototype.toInstanceArray = function (objs) {
+  if (!Array.isArray(objs)) {
+    objs = [objs];
+  }
+  return objs.map(function(obj) {
+    if (!(obj instanceof this.target.Instance)) {
+      var tmpInstance = {};
+      tmpInstance[this.target.primaryKeyAttribute] = obj;
+      return this.target.build(tmpInstance, {
```

Comment:
I'd say there's likely a good overlap between people contributing to oss node.js projects and people interested in ES6 syntax.

And if it improves the productivity of those who contribute most, i'd say it's likely worth it.
thoughts @overlookmotel @mbroadst?